src/chargebee.module-definition.ts

```diff
@@ -5,4 +5,14 @@ import { ChargebeeModuleOptions } from "./chargebee.interface";
 export const {
   ConfigurableModuleClass,
   MODULE_OPTIONS_TOKEN: CHARGEBEE_MODULE_OPTIONS_TOKEN,
-} = new ConfigurableModuleBuilder<ChargebeeModuleOptions>().build();
+} = new ConfigurableModuleBuilder<ChargebeeModuleOptions>()
+  .setExtras(
+    {
+      isGlobal: false,
```

Comment:
Can make this `true` by default to maintain compatibility.

```suggestion
      isGlobal: true,
```